routes/index.js

```diff
@@ -1,279 +1,43 @@
-'use strict'
-
-const { Writable } = require('stream')
-const {inspect} = require('util')
+const express = require('express')
+const id = require('./params/id')
+const type = require('./params/type')
 
 module.exports = (app) => {
-  // todo path.join will be needed for packaging with pkg
-
-  const path = require('path')
-  const index = path.join(__dirname, '../index.html')
-  const index2 = path.join(__dirname, '../index2.html')
-  const index3 = path.join(__dirname, '../index3.html')
-  const index4 = path.join(__dirname, '../index4.html')
-  const index5 = path.join(__dirname, '../index5.html')
-  const playerJs = path.join(__dirname, '../public/player.js')
-  const playerMinJs = path.join(__dirname, '../public/player.min.js')
-  const playerCss = path.join(__dirname, '../public/player.css')
+  app.disable('x-powered-by')
 
-  const websocket = path.join(__dirname, '../public/websocket.html')
-
-  app.all('/*', function (req, res, next) {
+  app.all('/*', (req, res, next) => {
     // todo verify authenticated and reroute if necessary
     // console.log('req.url', req.url)
     next()
   })
 
-  app.get(['/', '/index.html'], (req, res) => {
-    res.sendFile(index)
-  })
-
-  app.get('/websocket.html', (req, res) => {
-    res.sendFile(websocket)
-  })
-
-  app.get('/index2.html', (req, res) => {
-    res.sendFile(index2)
-  })
-
-  app.get('/index3.html', (req, res) => {
-    res.sendFile(index3)
-  })
-
-  app.get('/index4.html', (req, res) => {
-    res.sendFile(index4)
-  })
-
-  app.get('/index5.html', (req, res) => {
-    res.sendFile(index5)
-  })
-
-  app.get('/public/player.js', (req, res) => {
-    res.sendFile(playerJs)
-  })
-
-  app.get('/public/player.min.js', (req, res) => {
-    res.sendFile(playerMinJs)
-  })
-
-  app.get('/public/player.css', (req, res) => {
-    res.sendFile(playerCss)
-  })
-
-  app.param('id', (req, res, next, id) => {
-    // console.log('parse id')
-    const streams = app.locals.streams
-    const stream = streams.get(id)
-    if (!stream) {
-      return res.status(404)
-        .end(`stream "${id}" not found`)
-    }
-    if (!stream.hasOwnProperty('ffmpeg')) {
-      return res.status(500)
-        .end(`stream "${id}" does not have a valid ffmpeg src`)
-    }
-    res.locals.id = id
-    res.locals.stream = stream
-    next()
-  })
+  app.use(express.static('www'))
```

Comment:
I condensed a lot of your public routers with express's static middleware. No needs to have to manually send files. I left `compare.js` routing in the same style so you can have fun using `express.static()`. Just as a note, `static` is a reserved keyword, so we cannot do `const { static } = require('express')`